README.md

```diff
@@ -1,23 +1,28 @@
 pipl
 ==============
 
-Pipl search api wrapper written for rest api in node
+Pipl search api wrapper written for rest api in Node.
 
 ### Getting Started
 
-Just like others, its simple and quick as per standard:
+Just like others, it's simple and quick as per standard:
 
 `npm install pipl --save`
 
-this will install the module and add the entry in `package.json`. Lets start using it!
+This will install the module and add the entry in `package.json`. Lets start using it!
 
 ```javascript
-var pipl = require('pipl')('apiKey here');
+var pipl = require('pipl')('apiUrl', 'apiKey');
+```
+
+Example using v5 API url:
+```javascript
+var pipl = require('pipl')('http://api.pipl.com/search/v5/', 'apiKey');
```

Comment:
We are not changing the signature of the constructor because existing users only provide `apiKey`,  see below.